packages/api/src/utils/github/buildContributionQuery.ts

```diff
@@ -0,0 +1,38 @@
+export const buildQueryBodyString = (
+  projectsString: string,
+  dateString: string,
+  userString: string,
+  cursor: string | null,
+) => {
+  const queryParamString = `${projectsString} is:pr is:merged merged:${dateString} ${userString}`;
+
+  const queryBodyString = JSON.stringify({
```

Comment:
I would change this variable name to something as close to what it represents as possible so users won't have to read its full context to understand what it does. i.e. `pullRequestQuery`